src/test/java/org/jenkinsci/plugins/displayurlapi/actions/ActionRedirectExtendedTest.java

```diff
@@ -44,14 +58,35 @@ public void testRedirectForChangesURL() throws Exception {
             .header("Location", getRedirectedProvider().getChangesURL(run));
     }
 
+    @Test
+    public void testRedirectForTestsURL() throws Exception {
+        given()
+            .urlEncodingEnabled(false)
+            .redirects().follow(false)
```

Comment:
Hmm, do any of these tests actually use `JenkinsRule.WebClient` to verify that the HTTP request

* successfully follows all redirects and gets to an HTML page with a 200 status
* displays some sort of reasonable content, such as actual artifacts or test results

?

Reply:
No idea, this particular test framework was already in place and I did apply the same pattern than the one for the existing env variables. I'm not sure if those validations should be delegated to the upstream class, `ArtifactArchiver`. Do you think we do need to increase the test coverage for this?

I'd like to get this PR merged then https://github.com/jenkinsci/blueocean-display-url-plugin/pull/24 could be also released afterward

Reply:
> I'm not sure if those validations should be delegated to the upstream class, `ArtifactArchiver`.

No, because I am talking about verifying the behavior of the redirects specifically.

Reply:
Got it, for simplicity I did enable the redirect UTs for:
- artefacts
- job
- build

https://github.com/jenkinsci/display-url-api-plugin/pull/25/commits/d96f21b00c170db51bcb6b245253db995b2aa3dd

Thanks for the feedback Jesse